frontend/src/components/pages/ReviewPage.tsx

```diff
@@ -126,35 +169,67 @@ const ReviewPage = () => {
               setTranslatedStoryLines={setTranslatedStoryLines}
               numApprovedLines={numApprovedLines}
               setNumApprovedLines={setNumApprovedLines}
+              isReviewable={stage === "REVIEW"}
               reviewPage
             />
           </Flex>
-          <Flex margin="20px 30px" justify="flex-start" alignItems="center">
-            <Box marginRight="10px">
-              <ProgressBar
-                percentageComplete={
-                  (numTranslatedLines / translatedStoryLines.length) * 100
-                }
-                type="Translation"
-              />
-            </Box>
-            <Box>
-              <ProgressBar
-                percentageComplete={
-                  (numApprovedLines / translatedStoryLines.length) * 100
-                }
-                type="Review"
-              />
-            </Box>
+          <Flex margin="20px 30px" justify="space-between" alignItems="center">
+            <Flex justify="flex-start" alignItems="right">
+              <Box marginRight="10px" width="130px">
```

Comment:
rather than wrapping the `ProgressBar` with a box and setting the box's width, can you set the width in the `ProgressBar` component?